packages/amqp/lib/AbstractAmqpConsumer.ts

```diff
@@ -154,27 +154,26 @@ export abstract class AbstractAmqpConsumer<
         .then((result) => {
           if (result.result === 'success') {
             this.channel.ack(message)
-            this.handleMessageProcessed(originalMessage, 'consumed')
+            this.handleMessageProcessed(parsedMessage, 'consumed')
```

Comment:
Changed to always use parsed message on spy as it can break tests using `toEqual`